### DISCUSSION TITLE
Input and Select completely combined with Tablecell instead of nested

### DISCUSSION THREAD

**Discussion by chaocwu** [shadcn-ui/ui#4169] [Q&A]
How to make Input and Select completely combined with Tablecell instead of nested, nested will display their respective borders, which seems to be in the CELL aschild

**Comment by abhayclasher** [comment]
hey, just slap `className=\"border-none focus-visible:ring-0 px-0\"` on your Input or Select. then they'll just blend into the TableCell without those double borders. u might also want to add `p-0` to the TableCell itself so it fills the whole space 👍
